### DISCUSSION TITLE
Same document split screen

### DISCUSSION THREAD

**Discussion by Zineffable1** [sumatrapdfreader/sumatrapdf#2516] [Ideas]
So many times I'll be reading something and want to compare two blocks of text, two images or one of either, but, of course they are on different pages.

**Comment by GitHubRulesOK** [comment]
see
https://github.com/sumatrapdfreader/sumatrapdf/issues/2509#issuecomment-1112219971
 https://github.com/sumatrapdfreader/sumatrapdf/issues/1575

**Comment by kjk** [comment]
Use `Ctrl + Shift + N` to open second window with current document.
In pre-release build can also do `Ctrl + K` for command palette and pick `Open Current Document In New Window` command.

**Reply by Zineffable1** [reply]
cheers brudda
